lib/io/can.hpp

```diff
@@ -1,8 +1,15 @@
+#include <core/logger.hpp>
 #include <core/types.hpp>
 
 namespace hyped::io {
 class Can {
  public:
-  Can();
+  Can(hyped::core::ILogger &logger);
+  int sendCanFrame(can_frame message);
+  can_frame receiveCanFrame();
```

Comment:
This is an I/O operation. It can probably go wrong. Use `std::optional` to indicate that.